src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -348,22 +386,28 @@ export default React.memo(function GamePage(): JSX.Element | null {
                 <div className="developer">{developer}</div>
                 <div className="summary">{description}</div>
                 {is_installed && showCloudSaveInfo && (
-                  <div
+                  <p
                     style={{
                       color: autoSyncSaves ? '#07C5EF' : ''
                     }}
+                    className="iconWithText"
                   >
-                    <b>{t('info.syncsaves')}:</b>{' '}
+                    <CloudQueue />
+                    <b>{t('info.syncsaves')}</b>
+                    {': '}
                     {autoSyncSaves ? t('enabled') : t('disabled')}
-                  </div>
+                  </p>
                 )}
                 {is_installed && !showCloudSaveInfo && (
-                  <div
+                  <p
                     style={{
                       color: '#F45460'
                     }}
+                    className="iconWithText"
                   >
-                    <b>{t('info.syncsaves')}:</b>{' '}
+                    <CloudOff />
```

Comment:
I'm not sure why, but this icon gets focused when moving with a controller and can be clicked too
![image](https://user-images.githubusercontent.com/188464/219908792-325eadb4-9866-4363-8c31-65dc6573e098.png)

Reply:
Same happens with download size and install size
![image](https://user-images.githubusercontent.com/188464/219909073-77a7b527-d548-482b-9d4c-819bb6064c56.png)